controllers/utils/common.go

```diff
@@ -58,3 +58,10 @@ func NewSafeResourceName(name, suffix string, maxLength, spareLength int) string
 	}
 	return fmt.Sprintf("%s-%s", base, suffix)
 }
+
+// PolicyStatusUnknownError defines an error when policy status is missing or unknown
+type PolicyStatusUnknownError string
```

Comment:
I like this idea of having an error type with a message. In this case though, it might be a bit of an overkill. I would be ok with logging the different cases only before returning. Maybe just add this as another constant along with the existing ones in constants.go. The more interesting case is actually the non compliant case where the non compliant message could be useful for troubleshooting, if the policy just doesn't go compliant for a cluster. We will revisit this as part of https://issues.redhat.com/browse/CNF-3903.